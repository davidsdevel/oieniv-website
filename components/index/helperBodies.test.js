/*eslint-env jest*/
import React from "react";
import HelperBodies from "./helperBodies";
import {shallow} from "enzyme";
import renderer from "react-test-renderer";

describe("HelperBodies with Enzyme", () => {
	it("render eight HelperBody component", () => {
		const wrapper = shallow(<HelperBodies/>);
	
		expect(wrapper.find("HelperBody")).toHaveLength(7);
	});
});

describe("HelperBodies with Snapshot", () => {
	it("render without crash", () => {
		const component = renderer.create(<HelperBodies/>);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});
});
