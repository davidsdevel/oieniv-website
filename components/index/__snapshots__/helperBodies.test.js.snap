// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelperBodies with Snapshot render without crash 1`] = `
<div
  className="jsx-563012480"
  id="helper-bodies-container"
>
  <h3
    className="jsx-563012480 sub-titles"
  >
    Organos Auxiliares de OIENIV
  </h3>
  <p
    className="jsx-563012480"
  >
    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
  </p>
  <div
    className="helper-body"
  >
    <h4>
      Anjeiv
    </h4>
    <img
      alt="Organos Auxiliares - Anjeiv - OIENIV"
      src="/static/images/test-logo.png"
    />
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
    </p>
  </div>
  <div
    className="helper-body"
  >
    <h4>
      Andeiv
    </h4>
    <img
      alt="Organos Auxiliares - Andeiv - OIENIV"
      src="/static/images/test-logo.png"
    />
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
    </p>
  </div>
  <div
    className="helper-body"
  >
    <h4>
      Copaso
    </h4>
    <img
      alt="Organos Auxiliares - Copaso - OIENIV"
      src="/static/images/test-logo.png"
    />
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
    </p>
  </div>
  <div
    className="helper-body"
  >
    <h4>
      Pronani
    </h4>
    <img
      alt="Organos Auxiliares - Pronani - OIENIV"
      src="/static/images/test-logo.png"
    />
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
    </p>
  </div>
  <div
    className="helper-body"
  >
    <h4>
      Anjeiv
    </h4>
    <img
      alt="Organos Auxiliares - Anjeiv - OIENIV"
      src="/static/images/test-logo.png"
    />
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
    </p>
  </div>
  <div
    className="helper-body"
  >
    <h4>
      Andeiv
    </h4>
    <img
      alt="Organos Auxiliares - Andeiv - OIENIV"
      src="/static/images/test-logo.png"
    />
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
    </p>
  </div>
  <div
    className="helper-body"
  >
    <h4>
      Copaso
    </h4>
    <img
      alt="Organos Auxiliares - Copaso - OIENIV"
      src="/static/images/test-logo.png"
    />
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
    </p>
  </div>
  <div
    className="helper-body"
  >
    <h4>
      Pronani
    </h4>
    <img
      alt="Organos Auxiliares - Pronani - OIENIV"
      src="/static/images/test-logo.png"
    />
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
    </p>
  </div>
</div>
`;
